uat/codestyle/IntelliJ.xml

```diff
@@ -0,0 +1,69 @@
+<!--
```

Comment:
delete this.

Reply:
removed in 27fbe2119408cac23cefed5f76158c09e8a3cc20